bitwarden/server
# [PS-190] Add manual sync trigger in self hosted

## Type of change

## Objective

Adds manual sync button to Organization view of Admin project

Fixes many minor bugs with the sync


## Screenshots




![Screen Shot 2022-04-12 at 10 16 21 PM](https://user-images.githubusercontent.com/24985544/163859577-7e98010a-bcbd-43e0-89f7-04ec7c59cf37.png)
![Screen Shot 2022-04-12 at 10 41 12 PM](https://user-images.githubusercontent.com/24985544/163859593-06ccf5ad-d572-4ec5-89be-ffef2fc77f9f.png)

## Testing requirements

 - make sure Connections section only appears on self-hosted admin panel
 - make sure BillingSync does not show in connections if EnableCloudCommunication is disabled in global settings
 - manual sync button should be disabled after sync


## Before you submit
